meta-cube/recipes-support/overc-system-agent/files/overc-system-agent-1.2/Overc/overc.py

```diff
@@ -230,7 +230,15 @@ def _container_delete(self, container, template, force):
         self.message = self.container.message
 
     def container_upgrade(self):
-        self._container_upgrade(self.args.name, self.args.template, self.args.rpm, self.args.image)
+        # Perform overlay check fist
+        overlaylist = self.container.get_overlay(self.args.name)
+        if len(overlaylist)>0:
+            print "Container %s got overlayed dir, including" % self.args.name
+            print(overlaylist)
+            print "Must use system upgrade."
```

Comment:
s/Must use system upgrade./This container can only be upgraded via  a system upgrade/